### DISCUSSION TITLE
Prevent panning / repeating tiles

### DISCUSSION THREAD

**Discussion by mwolfeu** [vue-leaflet/vue-leaflet#357] [General]
How would I specify that I want to disable the repeating map by preventing panning beyond [ [-90, -180],[90, 180]] ?

Thanks in advance!

**Comment by 1stthomas** [comment]
I added following part in the setup script in the `onMounted` event:
```javascript
const bounds = L.latLngBounds(L.latLng(aa, ab), L.latLng(ba, bb));

lmap.value.leafletObject.setMaxBounds(bounds);
lmap.value.leafletObject.on('drag', function () {
  lmap.value.leafletObject.panInsideBounds(bounds, { animate: false });
});
```
Where `aa`, `ab` and `ba`, `bb` are the coordinates of your bounds and `lmap` is the reference to the `l-map` component.

(see: https://stackoverflow.com/questions/22155017/can-i-prevent-panning-leaflet-map-out-of-the-worlds-edge#answer-42961284)

**Reply by mwolfeu** [reply]
Perfect!  Thanks!
